python/ray/tests/test_actor.py

```diff
@@ -1446,6 +1446,47 @@ def hang(self):
         ray.get(result, timeout=1)
 
 
+# This test verifies actor creation task failure will not
+# hang the caller.
+def test_actor_creation_task_crash(ray_start_regular):
+     # Test actor death in constructor.
+     @ray.remote(max_reconstructions=0)
+     class Actor(object):
+         def __init__(self):
+             print("crash")
+             os._exit(0)
+
+         def f(self):
+             return "ACTOR OK"
+
+     a = Actor.remote()
+     try:
+         ray.get(a.f.remote())
+     except ray.exceptions.RayActorError:
```

Comment:
We should use with pytest.raises here, to assert that it raises an error.

Reply:
yep